tsrc/cli/review.py

```diff
@@ -0,0 +1,19 @@
+""" Entrypoint for Review """
+
+from .push import get_token, PushAction
+
+class ReviewAction(PushAction):
```

Comment:
nope. The relationship between ReviewAction and PushAction is _not_ is-a.

extract the functionality you need from PushAction and use composition instead of inheritance